src/ECAA80.c

```diff
@@ -23,8 +23,23 @@ void ECAA80_UnkFloAI_Chase(Evt* script, MobileAISettings* aiSettings, EnemyDetec
 
 #include "world/common/AddPlayerHandsOffset.inc.c"
 
-// ECAA80_ItemChoice_WaitForSelection (needs data?)
+// Needs data migrated
+#ifdef NON_MATCHING
+ApiStatus ECAA80_ItemChoice_WaitForSelection(Evt *script, s32 isInitialCall) {
+    Bytecode* args = script->ptrReadPos;
+    if (isInitialCall != 0) {
+        ECAA80_ItemChoice_HasSelectedItem = 0;
```

Comment:
this is a bool, so = 0 should be = FALSE and != 0 isn't necessary

Reply:
you resolved this, but the comment wasn't addressed here or in the following func

Reply:
to clarify, I am saying `ECAA80_ItemChoice_HasSelectedItem = 0;` should be `ECAA80_ItemChoice_HasSelectedItem = FALSE;`. and the != 0 check below should just be if (thing) {